pkg/queue/cluster_queue_interface.go

```diff
@@ -49,12 +49,15 @@ type ClusterQueue interface {
 	// queue is empty.
 	Pop() *workload.Info
 
-	// AddInadmissibleIfNotPresent inserts a workload that could not be
-	// admitted back into the ClusterQueue. The implementation might choose to
-	// keep it in temporary placeholder stage where it doesn't compete with
-	// other workloads, until cluster events free up quota. The workload should
-	// not be reinserted if it's already in the ClusterQueue.
-	AddInadmissibleIfNotPresent(*workload.Info) bool
+	// RequeueIfNotPresent inserts a workload that was not
+	// admitted back into the ClusterQueue. If the boolean is true,
+	// it means that we couldn't determine if the workload was admissible
+	// in the last cycle. If the boolean is false, the implementation might
```

Comment:
```suggestion
	// admitted back into the ClusterQueue. If the boolean is true,
	// the workloads should be put back in the queue immediately,
	// because we couldn't determine if the workload was admissible
	// in the last cycle. If the boolean is false, the implementation might
```